### PR TITLE
[9.0][FIX] sale_automatic_workflow: Add security groups to avoid show information always

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — COMMENTED** [review]
Why only sales manager?

**Comment by sergio-teruel** [comment]
Other repo modules changes:
- sale_exception

**Comment by sergio-teruel** [comment]
@pedrobaeza update for base.group_user

**Comment by pedrobaeza** [comment]
Well, I was thinking more on sales user, as employee doesn't see that objects.

File: sale_exception/views/sale_view.xml [hunk 1]
```diff
@@ -95,6 +96,7 @@
             <field name="name">sale_exception.view_order_tree</field>
             <field name="model">sale.order</field>
             <field name="inherit_id" ref="sale.view_order_tree"/>
+            <field name="groups_id" eval="[(4, ref('base.group_user'))]"/>
             <field name="arch" type="xml">
```
Comment (AaronHForgeFlow) [inline sale_exception/views/sale_view.xml:100]: It makes more sense to me to assign this to the sale user

File: sale_exception/views/sale_view.xml [hunk 2]
```diff
@@ -117,6 +120,7 @@
             <field name="name">sale_exception.view_sales_order_filter</field>
             <field name="model">sale.order</field>
             <field name="inherit_id" ref="sale.view_sales_order_filter" />
+            <field name="groups_id" eval="[(4, ref('base.group_user'))]"/>
             <field name="arch" type="xml">
```
Comment (AaronHForgeFlow) [inline sale_exception/views/sale_view.xml]: Same here

**Comment by pedrobaeza** [comment]
@sergio-teruel please change the user to finally merge this.

**Comment by rafaelbn** [comment]
Ping @sergio-teruel

**Review by rafaelbn — APPROVED** [review]
For a long time in production environment :+1:

**Comment by pedrobaeza** [comment]
There are still comments unattended.

File: sale_exception/views/sale_view.xml (hunk 1 above)
Comment (sergio-teruel) [reply sale_exception/views/sale_view.xml:100]: Change in this commit https://github.com/OCA/sale-workflow/pull/394/commits/aa1a7eb321e869448c3008326ecadfbf41149e12

File: sale_exception/views/sale_view.xml (hunk 2 above)
Comment (sergio-teruel) [reply sale_exception/views/sale_view.xml]: Idem. https://github.com/OCA/sale-workflow/pull/394/commits/aa1a7eb321e869448c3008326ecadfbf41149e12
:smiley:

**Comment by sergio-teruel** [comment]
@pedrobaeza I changed the group in this commit https://github.com/OCA/sale-workflow/pull/394/commits/aa1a7eb321e869448c3008326ecadfbf41149e12

File: sale_exception/views/sale_view.xml (hunk 1 above)
Comment (pedrobaeza) [reply sale_exception/views/sale_view.xml:100]: Well, this commit is not here in this PR...

**Comment by sergio-teruel** [comment]
@aheficent Fix security groups

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:

File: sale_exception/views/sale_view.xml [hunk 3]
```diff
@@ -69,6 +69,7 @@
             <field name="name">sale_exception.view_order_form</field>
             <field name="model">sale.order</field>
             <field name="inherit_id" ref="sale.view_order_form"/>
+            <field name="groups_id" eval="[(4, ref('base.group_sale_salesman'))]"/>
```
Comment (pedrobaeza) [inline sale_exception/views/sale_view.xml:72]: Why this one only on sales manager?

**Comment by yajo** [comment]
Travis is :x:

File: sale_exception/views/sale_view.xml (hunk 3 above)
Comment (yajo) [reply sale_exception/views/sale_view.xml:72]: I think you are mistaken.

Comment (pedrobaeza) [reply sale_exception/views/sale_view.xml:72]: Agh, incorrect line for commenting. It's in the above change the one that it's only on managers.

**Comment by rafaelbn** [comment]
@sergio-teruel could we fix travis and merge this please?

**Comment by victor-ditay** [comment]
Problem is with `cr.commit()` perhaps not in the guidelines of OCA by the time was merged

So it's here on the 9.0 branch https://github.com/OCA/sale-workflow/blob/9.0/sale_automatic_workflow/models/automatic_workflow_job.py#L29

**Comment by pedrobaeza** [comment]
Closing as this is out of interest now

**Pull request closed without merging**